ext/opcache/ZendAccelerator.c

```diff
@@ -3251,9 +3251,10 @@ static zend_result accel_post_startup(void)
 			if (JIT_G(buffer_size) == 0) {
 				JIT_G(enabled) = false;
 				JIT_G(on) = false;
-			} else if (!ZSMMG(reserved)) {
-				zend_accel_error_noreturn(ACCEL_LOG_FATAL, "Could not enable JIT: could not use reserved buffer!");
 			} else {
+				if (!ZSMMG(reserved)) {
```

Comment:
This change looks unnecessary.

Reply:
Thought this could improve readability a bit, reverting...